dashboard/modules/reporter/reporter_agent.py

```diff
@@ -186,12 +186,15 @@ def _get_workers(self):
 
     @staticmethod
     def _get_raylet_cmdline():
-        curr_proc = psutil.Process()
-        parent = curr_proc.parent()
-        if parent.pid == 1:
-            return ""
-        else:
-            return parent.cmdline()
+        try:
+            curr_proc = psutil.Process()
+            parent = curr_proc.parent()
+            if parent.pid == 1:
+                return []
```

Comment:
Can you briefly explain why you changed the default return value here? (Was it a bug before?)

Reply:
Yes, it was a bug. the `cmdline` returns a list instead of a string.